galaxy_ml/keras_model_config.xml

```diff
@@ -728,6 +728,23 @@
   <help>
       <![CDATA[
 **Help**
+
+**What it does**
+
+Creates a deep learning architecture using Keras deep learning library by adding different types of layers in a sequential manner. Different types of layers include Dense, Dropout, Convolutional, Activation and so on. These layers are added one after another which generates a deep architecture.
```

Comment:
Thanks a lot for adding 'help' text! This is great.
Have you tried making a `functional` model in the layers adding tool? The tool not only supports sequential model, but also functional model.